apps/api/v1/test/lib/event-types/_post.test.ts

```diff
@@ -108,10 +189,8 @@ describe("POST /api/event-types", () => {
           hidden: true,
           userId: memberUserId,
           parentId: 9999,
-          teamId: 9999,
         },
       });
-      req.isSystemWideAdmin = true;
```

Comment:
It was expecting checkParentEventOwnership to be called, but we are skipping it for systemWideAdmin, so it was removed here.